packages/components-js/tests/vrt/playwright/specs/custom/form-fields.vrt.ts

```diff
@@ -27,6 +27,7 @@ const scenario = async (page: Page, theme: Theme): Promise<void> => {
 
   const tagNameToChildMap: { [key in TagName]?: string } = {
     'p-checkbox-wrapper': '<input type="checkbox" />', // readonly is not supported
+    'p-pin-code': '', // readonly is not supported
```

Comment:
Would be more logical to put it next to the text-field.
Also is multi-select missing here?